workflowtrigger/util/clientrequest/request_events.go

```diff
@@ -1,7 +1,7 @@
 package clientrequest
```

Comment:
For all the serialization in these methods, order does not matter. Is it possible to automatically insert all the keys using the OrderedMap (not add them explicitly)?

Reply:
Unfortunately, OrderedMap does not have a method for bulk insertion. If we do not need those objects to be ordered we can use a simple map here. Updated those methods with `map[string]interface{}` return value.